.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,58 @@
+name: Bug Report
+description: File a bug report
+labels: ["Type: Bug", "Status: Triage"]
+body:
+  - type: markdown
+    attributes:
+      value: >
+        Thanks for taking the time to fill out this bug report! Before submitting your issue, please make
+        sure you are using the latest version of the charm. If not, please switch to this image prior to 
+        posting your report to make sure it's not already solved.
+  - type: textarea
+    id: bug-description
+    attributes:
+      label: Bug Description
+      description: >
+        If applicable, add screenshots to help explain your problem. If applicable, add screenshots to 
+        help explain the problem you are facing.
```

Comment:
```suggestion
        Provide an explanation of the issue you are facing. If applicable, add screenshots to help explain the problem.
```

Reply:
I do think "explanation" is a too general word. It can include logs, commands to reproduce or whatever but there are other sections for this. In your text I would just change "explanation" for "description".